### ISSUE TITLE
A11y_Azure Extensions for VS Code_ Create New Project _Screenreader: screenreader is not announcing combo box “select the folder that will contain your folder name”

### ISSUE THREAD

**Issue by Yash14j** [microsoft/vscode-azurefunctions#2229]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHub Tags:#A11yTCS;#Azure Tools for VS Code;#A11y_Azure Extensions for VS Code; #DesktopApp;#Win10;;#WCAG1.3.1;#Benchmark;#FTP;#STP;#NVDA;#JAWS;#A11yMAS;;#A11ySev2;#External: VSCode;

Environment Details:
Application Name: Azure Extensions for VS Code
Version: 1.46.1 (user setup)
Commit: cd9ea6488829f560dc949a8b2fb789f3cdc05f5d
Date: 2020-06-17T21:13:20.174Z
Electron: 7.3.1
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
Windows Version: Windows10

**Repro Steps:**

1. Open Visual studio code 
2. Click on extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button
4. Open screenreader and click on "Create New Project Button" from the toolbar 
5. Create new project pop will appear
6.After popup appears press tab key and then press shift+tab key to check the result.

**Actual:**

When we press tab key and then again press shift+tab key screenreader does not announces name of combo box “select the folder that will contain your folder name”

**This issue is also observed in Create Function and Deploy to function app screen 
Same issue is also observed with both NVDA and JAWS.**

**Expected:**

Screenreader should announce name of combo box “select the folder that will contain your folder name” when focus comes on it

**User Impact:**

User will not get proper information about the combo box 

**Attachment:**
[#2229.zip](https://github.com/microsoft/vscode-azurefunctions/files/4958045/2229.zip)

**Comment by EricJizbaMSFT** [comment]
@Yash14j I can't seem to repro this. I tried NVDA and it was announcing "Select the folder..." when I first entered the combo box and after tabbing and then shift tabbing to get back. I tried to watch the attached video to get more info, but I can't get it to work. I get an error like this:
<img width="359" alt="Screen Shot 2020-07-20 at 11 35 06 AM" src="https://user-images.githubusercontent.com/11282622/87986082-7cbc3800-ca91-11ea-886c-6bfffef3ac21.png">

Any ideas?

**Comment by Yash14j** [comment]
@EricJizbaMSFT this issue still repro in the Visual Studio Code version: 1.49.0, extension version 0.24.0.
**REPRO STEPS**:
1.  Open Visual studio code
2.  Click on extension button and search for "Azure Functions"
3.  Click on install and move to the "Azure" button
4.. Open screenreader and click on "Create New Project Button" from the toolbar
5.  Create new project pop will appear
6.  Press tab key and then press down arrow key.
7.  Focus is on "browse". Now press "shift+tab key" to check the result

Please refer the attachment for the same.
[2229_REPRO.zip](https://github.com/microsoft/vscode-azurefunctions/files/5223738/2229_REPRO.zip)

**Comment by EricJizbaMSFT** [comment]
This is a bug in VS Code. I can repro it with the following steps (aka without our extension), and I've listed the NVDA speech viewer logs below each step

1. Open the [command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette)
    > Type the name of a command to run.  combo box  has auto complete  editable
    > \>
1. Tab to quick pick list
    > Quick Input  list
1. Down arrow to first command
    > Tasks: Run Build Task, Control+Shift+B  1 of 718
1. Shift-tab back to input box
    *This should repeat the same thing as the first step, but it doesn't*

**Comment by Yash14j** [comment]
As per the C & AI bug process assigning the bug to the Product Owner. Please assign this bug back to the tester once the "External" issue has been resolved and changes are pushed to the environment that will be available for bug verification.
[https://github.com/microsoft/vscode/issues/108297](https://github.com/microsoft/vscode/issues/108297)

**Comment by EricJizbaMSFT** [comment]
Closing since this bug is external and due to lack of response from accessibility test team on how to get this off our tracking list.
